bin/build-styles

```diff
@@ -10,7 +13,39 @@ const inputFile = './src/assets/css/styles.scss';
 const outputFile = './build/blog/assets/css/styles.css';
 const isProduction = process.env.ELEVENTY_ENV === 'production';
 
+const buildAddonsFrontendBlogUtils = async function () {
+  const addonsFrontendBlogUtilsInputFile = path.resolve(
+    'node_modules/addons-frontend-blog-utils/style.css'
+  );
+  const addonsFrontendBlogUtilsOutputFile = path.resolve(
+    'build/blog/assets/css/addons-frontend-blog-utils.style.css'
+  );
+  const assetsPath = path.resolve('build/blog/assets/img/');
+  const css = await fs.readFile(addonsFrontendBlogUtilsInputFile, 'utf8');
+  // addons-frontend-blog-utils style.css has paths to images, we want to copy
+  // those to the right directory, rewrite the stylesheet, then the rewritten
+  // file can be imported from ours.
+  const options = {
+    url: 'copy',
+    basePath: path.resolve('node_modules/addons-frontend-blog-utils'),
+    assetsPath,
+    useHash: true,
+  };
+
+  await fs.mkdir(assetsPath, { recursive: true });
```

Comment:
Just curious about why you're using `await fs.mkdir` instead of `fs.mkdirSync`?

Reply:
Consistency with the rest of the scripts in addons-blog that are written that way